{easyrpg-player,liblcf}: 0.5.4 -> 0.6.0

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/liblcf/versions.



meta.description for liblcf is: '"Library to handle RPG Maker 2000/2003 and EasyRPG projects"'.

[Release on GitHub](https://github.com/EasyRPG/liblcf/releases/tag/0.6.0)


[Compare changes on GitHub](https://github.com/EasyRPG/liblcf/compare/0.5.4...0.6.0)

<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 0.6.0 with grep in /nix/store/0860vg2mdaa804kcsxgwsxslmamgl354-liblcf-0.6.0
- directory tree listing: https://gist.github.com/642c2e4381647a870b2ac578fa86f243
- du listing: https://gist.github.com/d7f92cfed5184626ccc706073e59f00f

</details>
<details>
<summary>
Rebuild report (if merged into master) (click to expand)
</summary>

6 total rebuild path(s)

2 package rebuild(s)

2 x86_64-linux rebuild(s)
2 i686-linux rebuild(s)
0 x86_64-darwin rebuild(s)
2 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
easyrpg-player
liblcf

</details>

<details>
<summary>
Instructions to test this update (click to expand)
</summary>

Either download from Cachix:
```
nix-store -r /nix/store/0860vg2mdaa804kcsxgwsxslmamgl354-liblcf-0.6.0 \
  --option binary-caches 'https://cache.nixos.org/ https://r-ryantm.cachix.org/' \
  --option trusted-public-keys '
  r-ryantm.cachix.org-1:gkUbLkouDAyvBdpBX0JOdIiD2/DP1ldF3Z3Y6Gqcc4c=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(r-ryantm's Cachix cache is only trusted for this store-path realization.)

Or, build yourself:
```
nix-build -A liblcf https://github.com/r-ryantm/nixpkgs/archive/cbae7a2bbd156fcda89ff03a2fa9bddf95c0a674.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/0860vg2mdaa804kcsxgwsxslmamgl354-liblcf-0.6.0
ls -la /nix/store/0860vg2mdaa804kcsxgwsxslmamgl354-liblcf-0.6.0/bin
```


</details>
<br/>


cc @yegortimoshenko for testing.

r-ryantm committed:
liblcf: 0.5.4 -> 0.6.0

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/liblcf/versions

timokau reviewed (changes requested):
This [breaks](http://sprunge.us/LpHKtl) `easyrpg-player` according to `nix-review`.

c0bw3b committed:
easyrpg-player: 0.5.4 -> 0.6.0

Changelog:
https://blog.easyrpg.org/2019/03/easyrpg-player-0-6-0-preemptive-attack/

c0bw3b reviewed (approved):
Both pkg needs to be updated together
